perception/lidar_apollo_segmentation_tvm/package.xml

```diff
@@ -13,7 +13,7 @@
   <build_depend>autoware_cmake</build_depend>
   <build_depend>libpcl-all-dev</build_depend>
 
-  <depend>autoware_auto_common</depend>
+  <depend>autoware_auto_geometry</depend>
```

Comment:
The dependency on `autoware_auto_geometry` was removed by #3165 (and broke the package because I didn't test that it worked at the time), and #3384 adds the dependency that is actually needed (`tf2_geometry_msgs`). Can we get #3384 merged first and then rebase this on it?

Reply:
Ok, merged https://github.com/autowarefoundation/autoware.universe/pull/3384

Rebased and removed `autoware_auto_geometry` with: https://github.com/autowarefoundation/autoware.universe/pull/3136/commits/21b629b2fe8458187a9bec0ea19c4c46d61d5b10